README.md

```diff
@@ -37,3 +37,10 @@ npm install -g webpack
 npm install
 ```
 
+
+// database login
+// scripts
+// install or run webpack if its not working
+// port numbers
+// endpoints
```

Comment:
I assume these are placeholders for instructions that will be added in a later pull request?

Reply:
Yup need to fill these our